appserver/ejb/ejb-container/src/main/java/com/sun/ejb/containers/StatelessSessionContainer.java

```diff
@@ -759,6 +772,8 @@ public void destroy(Object obj) {
 
     private static class PoolProperties {
         int maxPoolSize;
+        int maxWaitTimeInMillis = Integer.getInteger(String.format("fish.payara.ejb-container.%s", RuntimeTagNames.MAX_WAIT_TIME_IN_MILLIS),
```

Comment:
No. this field already exists but marked deprecated. I think wE do need to document it as it works a bit differently now.